api/src/main/resources/workbench-api.yaml

```diff
@@ -804,6 +804,20 @@ paths:
           description: Workspace deletion request accepted
           schema:
             "$ref": "#/definitions/EmptyResponse"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/get-data-page-items":
```

Comment:
We don't need `get` in the resource name, since that's in the verb.

Also, can we be more descriptive than "data page"? We don't need `items` either, as it doesn't really clarify what we're fetching.

Reply:
Talked offline! Renaming to `cdr-selectors`